react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class HotButton extends React.Component {
```

Comment:
For future exercises you want to put your component in its own file and export it, then import it into `index.jsx`